From 6faa3de2d0f30dd0595fbb1b09eb87559774a576 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 23 Dec 2017 23:21:56 +0200
Subject: [PATCH 145/249] lineage: Remove external/koush/ion

* It was used in VoicePlus and some cyngn apps,
  but now it's unused, so kill it

Change-Id: I7236f22987c1d65f2f4286cf4ef4411fd70f0998
---
 snippets/lineage.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 267f631..86270db 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -9,7 +9,6 @@
   <!--<project path="external/ffmpeg" name="LineageOS/android_external_ffmpeg" />-->
   <project path="external/fuse" name="LineageOS/android_external_fuse" />
   <project path="external/htop" name="LineageOS/android_external_htop" />
-  <!--<project path="external/koush/ion" name="LineageOS/ion" />-->
   <project path="external/libncurses" name="LineageOS/android_external_libncurses" />
   <project path="external/libnfc-nxp" name="LineageOS/android_external_libnfc-nxp" />
   <project path="external/libtar" name="LineageOS/android_external_libtar" />
-- 
2.17.1

